frontends/ol-components/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react"
+import { PlatformEnum } from "api"
+
+export const PLATFORMS = [
+  {
+    code: PlatformEnum.Ocw,
+    name: "OCW",
+    image: "mit-ocw-logo-square.png",
+  },
+  {
+    code: PlatformEnum.Edx,
+    name: "edX",
+    image: "edx_logo.png",
```

Comment:
These are using existing image in the codebase where available, though some are low quality and don't lend themselves to the design format for the learning resource drawer.

Do we have a set of these with a fixed aspect ratio, or could we put together? 2:1 to 3:1 seems about right. @steven-hatch


<img width="165" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/782e0a5b-c3fc-4517-887f-703b358281ef">

<img width="172" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/202f0f6a-c2dc-4390-9d91-b3a5a7bd9549">

<img width="182" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/92a751eb-d198-4b34-b549-a825a91134e3">

Reply:
Incidentally, I think that's the old edX logo. I believe they use 

<img style="width:100px" src="https://www.edx.org/images/logos/edx-logo-elm.svg" alt="edx logo" />

now

Reply:
2:1 to 3:1 seems right. @mbilalmughal feel free to weigh in but the idea is to get the height consistent between all logos. This height should be the same as the button found on same line, to the right. If a logo occupies the height and runs into other elements then we would need to reduce width. I dont see this happening, but OpenCourseWare would be the one to give us trouble because its so wide. 

Also, please let me know if you need clean SVGs for logos. We have access to MIT Brand files for logos.